Agents/HistoricalNTUEnergyAgent/README.md

```diff
@@ -0,0 +1,190 @@
+
+The following shows the content of compatible Excel file for NTU Energy:
+![Shows part of the response body of a successful energy readings request.](docs/img/example_energy_readings.png "Energy readings general")
+
+#### Example readings
+Readings are retreived from the Excel file in the form of a JSON array with one JSON object per timestamp. The following
+shows a single JSON object example for Energy readings:
+
+```
+"GENERATOR_NODE_P_KW"=0
+"GENERATOR_NODE_Q_KVAR"=0
+"CONNECTION_NODE_P_KW"=0
+"CONNECTION_NODE_Q_KVAR"=0
+"NEC_P_KW"=0.029759984
+"NEC_Q_KVAR"=0.005736001
+"CANTEEN_2_P_KW"=0.020969993
+"CANTEEN_2_Q_KVAR"=0.0039
+"SPMS_P_KW"=0.26620004
+"SPMS_Q_KVAR"=0.106099997
+"RTP_P_KW"=0.04212
+"RTP_Q_KVAR"=0.018470005
+"N1_3_P_KW"=0.07400001
+"N1_3_Q_KVAR"=0.0183
+"N_2_P_KW"=0.022709971
+"N_2_Q_KVAR"=0.004809972
+"N_2_1_P_KW"=0.018199989
+"N_2_1_Q_KVAR"=0.007699982
+"SBS_P_KW"=0.21566999
+"SBS_Q_KVAR"=0.053040015
+"PIONEER_HALL_P_KW"=0.016668339
+"PIONEER_HALL_Q_KVAR"=0.002011665
+"THE_WAVE_P_KW"=0.002389999
+"THE_WAVE_Q_KVAR"=-0.000520001
+"HALL_4_P_KW"=0.011159994
+"HALL_4_Q_KVAR"=-0.003280001
+"EMB_P_KW"=0.101707703
+"EMB_Q_KVAR"=4.02891E-05
+"NYA_P_KW"=0.002770001
+"NYA_Q_KVAR"=0.000400001
+```
+These readings represent the values of active and reactive power for different locations in the NTU energy system.
+Active power (P) is measured in kilowatts (KW) and represents the power that is actually used by the electrical 
+equipment to perform useful work, while reactive power (Q) is measured in kilovolt-ampere reactive (KVAR) and 
+represents the power that is used to establish and maintain the electric and magnetic fields in the equipment.
+
+# 2. Setup and Running
+This part of the README describes the setup of the agent. The module itself can be packaged into an executable war, deployed as
+a web servlet on tomcat. Sending the appropriate request to the correct URL will initiate the agent. Since it uses the time-series
+client which maintains both instances in a knowledge graph and a Postgres database to store the data, these will be required to be
+set-up beforehand.
+
+## 2.1 Agent Setup
+It is required to have several config files set up before running the agent. In specific, one mapping file, two property files, and one xlsx file are required:
+- The [mapping file](#mapping-files) contains the mapping between the keys in the Excel file and the IRI of the KG instances.
+- The [agent.properties](#agent-properties) file points to the mapping configuration.
+- One [xlsxconnector.properties](#excel-connector-properties) file defines the number of columns/keys for the power readings Excel file.
+- One data file for energy readings in Excel format needs to be named `NTU_Energy_Consumption.xlsx` and included in the `config` folder. An example data file can be found at HistoricalNTUEnergyAgent\data\NTU_Energy_Consumption_Example.xlsx.
```

Comment:
example data file missing
and instead of .xlsx perhaps the example should be a .csv

Reply:
Fixed. The example .xlsx is replaced by an example screenshot in README.